packages/lesmis/src/components/LocationHeader.js

```diff
@@ -123,15 +124,15 @@ export const LocationHeader = () => {
             <ToggleButton
               value="dashboard"
               component={Link}
-              to={makeEntityLink(entityCode, 'dashboard')}
+              to={`${makeEntityLink(entityCode, 'dashboard')}${search}`}
```

Comment:
add the url params in the dashboard and map tab links so that it remembers where you are when navigating between the tabs

Reply:
And I guess this can't live within `makeEntityLink` because that's used in other cases where we want to clear the search?

Reply:
Yes exactly